### DISCUSSION TITLE
Support for rich-text features

### DISCUSSION THREAD

**Discussion by ilCosmico** [mozts2005/ZendeskApi_v2#509] [Q&A] [answered]
Any chance to add the support for the new rich-text-editing features via API?

https://support.zendesk.com/hc/en-us/articles/360057334013-Help-Center-adds-Rich-Text-Editing-to-your-Submit-a-Request-forms

![image](https://user-images.githubusercontent.com/8678145/111596509-87200480-87cd-11eb-95fb-98ccb1e32e6f.png)

I mean a way to tag and format blocks of source code when submitting tickets, like 
<code>
private void MyMethod()
{
    ...
}
</code>
or some kind of markdown for **Bold** or _Italic_ texts.

**Comment by mozts2005** [comment, marked as answer]
This would be a question that they Zendesk support team/ forums would be able to help with.

**Reply by ilCosmico** [reply]
I found this article: https://developer.zendesk.com/apps/docs/support-api/ticket_editor#invoke
Can it be used with this client?

**Reply by mozts2005** [reply]
this is part of the Apps Support API which is not supported via this library.

This Library is intended to be used as a backend wrapper to the Zendesk REST APIs not as a UI to Zendesk.

**Reply by ilCosmico** [reply]
It's a pity, but thanks for the reply.
